src/core/MOM_dynamics_split_RK2.F90

```diff
@@ -154,8 +154,7 @@ module MOM_dynamics_split_RK2
   logical :: split_bottom_stress  !< If true, provide the bottom stress
                                   !! calculated by the vertical viscosity to the
                                   !! barotropic solver.
-  logical :: calc_dtbt            !< If true, calculate the barotropic time-step
-                                  !! dynamically.
+  logical :: dtbt_use_bt_cont = .false. !< If true, use BT_cont to calcullate DTBT.
```

Comment:
`CS%dtbt_use_bt_cont` should only be initialized via a `get_param()` call, so there should not be any reason to initialize it on the declaration line.  Also, in this comment, "calcullate" should be "calculate".  The same comment applies to line 160 of MOM_dynamics_split_RK2b.F90.

Reply:
Done.